lib/DDGC/DB/Result/InstantAnswer.pm

```diff
@@ -395,6 +395,53 @@ many_to_many 'users', 'instant_answer_users', 'user';
 has_many 'instant_answer_topics', 'DDGC::DB::Result::InstantAnswer::Topics', 'instant_answer_id';
 many_to_many 'topics', 'instant_answer_topics', 'topic';
 
+after insert => sub {
+    my ( $self ) = @_;
+    my $schema = $self->result_source->schema;
+    $schema->resultset('ActivityFeed')->create(
+        $self->ddgc->config->subscriptions->created_ia_page( {
```

Comment:
Why not have the defaults here?  Probably because they are used in more than one place? If so, then how about adding a method to `DDGC::DB::Result::ActivityFeed` called .. `create_for_new_ia_page()` (there must be a better name but I can't think of one).
```
$schema->resultset('ActivityFeed')->create_for_new_ia_page({meta1 => ..., meta2 => ..., description => ...});
```